### PR TITLE
feat(atomic): facet component

### PR DESCRIPTION
**Pull request by nrawji**
KIT-463
https://coveord.atlassian.net/browse/KIT-463

### CODE REVIEW

**Review by olamothe — APPROVED** [review]

File: packages/atomic/src/locales.json [hunk 1]
```diff
@@ -7,6 +7,10 @@
     "en": "Insert a query. Press enter to send.",
     "fr": "Insérer une requête. Appuyez sur Entrée pour envoyer."
   },
+  "facetSearch": {
```
Comment (olamothe) [inline packages/atomic/src/locales.json:10]: I think we could just use the normal `search` entry here ?

File: packages/atomic/src/locales.json [hunk 2]
```diff
@@ -15,6 +19,12 @@
     "en": "Clear",
     "fr": "Effacer"
   },
+  "showMore": {
+    "en": "show more"
```
Comment (olamothe) [inline packages/atomic/src/locales.json:23]: Capitalized ?
in french: `Voir plus de résultats`

File: packages/atomic/src/locales.json [hunk 3]
```diff
@@ -15,6 +19,12 @@
...
+  },
+  "showLess": {
+    "en": "show less"
```
Comment (olamothe) [inline packages/atomic/src/locales.json:26]: In french: `Voir moins de résultats`.

You can also copy paste them from `strings.json` in JSUI repo.

Comment (ThibodeauJF) [reply packages/atomic/src/locales.json:26]: Yes there is possibly more languages for these strings already

**Review by ThibodeauJF — APPROVED** [review]

**Pull request merged**